gt/main.c

```diff
@@ -795,12 +1008,13 @@ gt_proc(void *arg)
 			continue;
 
 		for (i = 0; i < num_rx; i++) {
-			int ret;
 			struct rte_mbuf *m = rx_bufs[i];
 			struct gt_packet_headers pkt_info;
 			struct ggu_policy policy;
 			struct rte_mbuf *notify_pkt;
 
+			death_row.cnt = 0;
```

Comment:
This initialization can go before the `while` loop.

Sorry for commenting again on this initialization, but the code here is still changing.